content/en/user-manual/packs/loading-scenes.md

```diff
@@ -8,17 +8,46 @@ position: 3
 
 This page will take you through loading scenes with code and also some different approaches of using scenes in projects.
 
-There are two main approaches in using scenes:
+There are two main approaches in using scenes: changing scenes completely and additively loading scenes.
 
-### Switching scenes completely
+## Changing scenes completely
 
 This is the most common approach that developers take where each scene is a self-contained part of the game. For example, one scene would be the title screen and then one scene per level.
 
-[Here is an example][switch-scenes-completely-project] where the user can move to and from the title screen to other levels. (Please note that it's using the recommended practice below for loading scenes).
+[Here is an example][switch-scenes-completely-project] where the user can move to and from the title screen to other levels.
 
 <iframe loading="lazy" src="https://playcanv.as/e/p/Q1gKd1ek/" title="Switching Scenes Completely"></iframe>
 
-### Loading scenes additively
+This is down by simply calling [`SceneRegistry.changeScene`][changescene-api] with the name of the scene.
+
+```
+this.app.scenes.changeScene('Some Scene Name');
+```
+
+If the scene data is not already loaded, this function will:
+
+- Make the asynchronous network request for the new scene data.
+- When the scene data is loaded, it will delete all child entities from the application root node (destroying the existing scene hierarchy).
+- Call `loadSceneSettings` which is now synchronous as the scene data is loaded.
+- Call `loadSceneHierarchy` which is now synchronous as the scene data is loaded.
+
+If you want to know when the scene is loaded or if there are errors, you will need to provide a callback:
+
+```
+this.app.scenes.changeScene('Some Scene Name', (err, loadedSceneRootEntity) {
+    if (err) {
+        console.error(err);
+    } else {
+        // Scene hierachary has successfully been loaded
+    }
+});
+```
+
+To avoid the asynchronous network request for the new scene data at the point of calling `changeScene`, you can call [`SceneRegistry.loadSceneData`][loadscenedata-api] ahead of time and `changeScene` will become a synchronous call and immediately calls `loadSceneSettings` and `loadSceneHierarchy`.
```

Comment:
```suggestion
To avoid the asynchronous network request for the new scene data at the point of calling `changeScene`, you can call [`SceneRegistry.loadSceneData`][loadscenedata-api] ahead of time and `changeScene` will become a synchronous call that immediately calls `loadSceneSettings` and `loadSceneHierarchy`.
```